components/rating/stories/template.js

```diff
@@ -65,9 +65,8 @@ export const Template = ({
 					<span
 						class=${classMap({
 							[`${rootClass}-icon`]: true,
-							"is-selected": !isDisabled && idx <= value - 1,
-							"is-currentValue":
-								!isDisabled && !isReadOnly && idx === value - 1,
+							"is-selected": idx <= value - 1,
+							"is-currentValue": idx === value - 1,
```

Comment:
What was the purpose of having the `isDisabled` and `isReadOnly` args? I don't quite understand the `idx` bits, so that might be part of my problem.

Also (because I couldn't drop a comment on the correct lines)- I don't know that `readonly` is available to range inputs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#additional_attributes

Reply:
I've added some docs to the main component description that explains the use of those two classes.

Good callout on the `readonly` attribute being used in this template. I've added a commit that removes that, since it is not supported in HTML. That's popped up in a couple components now.